python/options.py

```diff
@@ -0,0 +1,55 @@
+import argparse
```

Comment:
Let's also rename this file to not be ambiguous.  There is `$GEM_BASE/python/AnalysisOptions.py` file already.  This file should be renamed to `$GEM_BASE/python/PlotOptions.py`.  To do this try using:

git mv options.py PlotOptions.py